### DISCUSSION TITLE
offline install

### DISCUSSION THREAD

**Discussion by zapewne** [markqvist/Reticulum#483] [Ideas]
What install all program rnsa, rnsx etc. without internet on computer?

**Comment by faragher** [comment]
You'd need to install Python and all the packages and their dependencies. That's beyond my knowledge, but if you're familiar with Python and PyPl, you could probably walk through the packages and download the requirements manually, put it on removable storage, and manually install on the offline system.

**Comment by zirpu** [comment]
There's also https://pypi.org/project/pex/ for a self-contained deployment binary.

**Reply by zapewne** [reply]
this project need internet too.
sorry , but I cant put program on flash drive and give my friend in emergency situation for increase network

**Reply by faragher** [reply]
https://www.slingacademy.com/article/python-ways-to-install-packages-offline/

Solution 3 is probably what you'd need.
